build/install-dotnet.yml

```diff
@@ -0,0 +1,14 @@
+steps:
+- task: UseDotNet@2
+  inputs:
+    packageType: 'sdk'
+    version: '3.1.x'
+    installationPath: 'C:\Program Files\dotnet'
```

Comment:
this line makes it work... previously it was installing to `C:\hostedtoolcache\windows\dotnet`, which suddenly stopped working